includes/activity/class-base-object.php

```diff
@@ -622,9 +630,11 @@ public function from_array( $array ) {
 	 * It tries to get the object attributes if they exist
 	 * and falls back to the getters. Empty values are ignored.
 	 *
+	 * @param bool $include_json_ld_context
+	 *
 	 * @return array An array built from the Object.
 	 */
-	public function to_array() {
+	public function to_array( $include_json_ld_context = true ) {
```

Comment:
I am not sure this is a good idea, because we have to add this var to every method that calls `to_array`, like `to_json` for example... Is there an easier, more elegant way?

Maybe re-add a class attribute like `private $json_ld_context;` that can be set to false?

I would love to see something like that, to have an easy way to extend it through code, like the `set_json_ld_context` (replaces the const) or `add_json_ld_context` (extends the const).